tunnel/tunnel.go

```diff
@@ -102,16 +98,14 @@ type Tunnel struct {
 
 // New creates a new instance of Tunnel.
 func New(localAddress string, server *Server, remoteAddress string) *Tunnel {
-
-	if localAddress == "" {
-		localAddress = "127.0.0.1:0"
-	} else if strings.HasPrefix(localAddress, ":") {
-		localAddress = fmt.Sprintf("127.0.0.1%s", localAddress)
+	cfg, err := NewSSHConfigFile()
+	if err != nil {
+		log.Debug("error to read ssh config")
```

Comment:
* Change the log level to warning since it is an error but the application will continue to work
* Adding the invalid configuration to the log message

```suggestion
		log.Warningf("error to read ssh config: %v", err)
```